src/main/java/seedu/address/security/SecurityManager.java

```diff
@@ -38,34 +41,44 @@ public boolean getAuthentication() {
 
     @Override
     public void login(String username, String password) {
-        if (username.equals(this.username) && password.equals(this.password)) {
-            this.isAuthenticated = true;
-            //Instantiates User in Model manager
-            model.matchUserToPerson(username);
-            //TODO Implement logger
-            //System.out.println("Correct Password");
-            raise(new SuccessfulLoginEvent());
-        } else {
-            //TODO Raise an exception to prompt user to type again
-            //TODO implement logger
-            //System.out.println("Incorrect password");
-            raise(new UnsuccessfulLoginEvent());
+        userlist = appUsers.getAccountCredentials();
+        for (AccountCredential acc : userlist) {
```

Comment:
Maybe not the fastest implementation? Seems like it'd be faster using a HashMap of sorts (O(1) rather than O(N))...

Wait wrong module oops

Reply:
Lol I knew you would say this. I'm just worried about jsonutil does not know how to parse back into a hashmap, can put this as V1.4